src/Billing/Controllers/BitPayController.cs

```diff
@@ -66,10 +66,16 @@ public async Task<IActionResult> PostIpn([FromBody] BitPayEventModel model, [Fro
             }
 
             var invoice = await _bitPayClient.GetInvoiceAsync(model.Data.Id);
-            if (invoice == null || invoice.Status != "confirmed")
```

Comment:
Is there anything in place to prevent duplicate invoices from being submitted? There's no unique index or constraint on `dbo.Transaction` (not sure if that's a big deal or not)

Reply:
Yes, see down a few lines. It will skip if invoice is already in DB.

Reply:
I looked but must have missed it. Thanks!